opensource/project/test-and-docs.md

```diff
@@ -239,99 +198,120 @@ Git for Windows installation.  **Git Bash**, just as it sounds, allows you to
-documentation. Of course, you don't need to install this generator
-to build the documentation, it is included with container.
+The Docker documentation source files are in a centralized repository at
+https://github.com/docker/docker.github.io. The content is
+written using extended Markdown, which you can edit in a plain text editor such
+Atom or Notepad. The docs are built using [Jekyll](https://jekyllrb.com/).
 
-You should always check your documentation for grammar and spelling. The best
-way to do this is with <a href="http://www.hemingwayapp.com/"
-target="_blank">an online grammar checker</a>.
+Most documentation is developed in the centralized repository. The exceptions are
+a project's API and CLI references and man pages, which are developed alongside
+the project's code and periodically imported into the documentation repository.
+
+Always check your documentation for grammar and spelling. You can user
+an online grammar checker such as http://www.hemingwayapp.com/ or a spelling or
+grammar checker built into your text editor. If you spot spelling or grammar errors,
+fixing them is one of the easiest ways to get started contributing to opensource
+projects.
 
 When you change a documentation source file, you should test your change
 locally to make sure your content is there and any links work correctly. You
-can build the documentation from the local host. The build starts a container
-and loads the documentation into a server. As long as this container runs, you
-can browse the docs.
-
-1. In a terminal, change to the root of your `docker-fork` repository.
-
-        $ cd ~/repos/docker-fork
-
-2. Make sure you are in your feature branch.
-
-        $ git status
-        On branch dry-run-test
-        Your branch is up-to-date with 'origin/dry-run-test'.
-        nothing to commit, working directory clean
-
-3. Build the documentation.
-
-        $ make docs
-
-    When the build completes, you'll see a final output message similar to the
-    following:
-
-        Successfully built ee7fe7553123
-        docker run --rm -it  -e AWS_S3_BUCKET -e NOCACHE -p 8000:8000 "docker-docs:dry-run-test" mkdocs serve
-        Running at: http://0.0.0.0:8000/
-        Live reload enabled.
-        Hold ctrl+c to quit.
-
-4. Enter the URL in your browser.
-
-    If you are using Docker Machine, replace the default localhost address
-    (0.0.0.0) with your DOCKERHOST value. You can get this value at any time by
-    entering `docker-machine ip <machine-name>` at the command line.
-
-5. Once in the documentation, look for the red notice to verify you are seeing the correct build.
-
-    ![Beta documentation](images/red_notice.png)
-
-6. Navigate to your new or changed document.
-
-7. Review both the content and the links.
-
-8. Return to your terminal and exit out of the running documentation container.
+can build the documentation from the local host.
+
+### Building the docs for docs.docker.com
```

Comment:
I think you're right, my feedback is not a blocker so let's merge this and iterate once we have more feedback.

About people that might want to use Jekyll in other repos, I think it's the other way around. By using docker we're lowering the barrier to contribute with docs. Of course there are people that might want to use Jekyll in multiple projects, but those are not the majority.